webviz_subsurface/plugins/_tornado_plotter_fmu/shared_settings/_view_settings.py

```diff
@@ -0,0 +1,196 @@
+                clearable=False,
+            ),
+            wcc.SelectWithLabel(
+                id=self.register_component_unique_id(ViewSettings.IDs.SENSITIVITEIS),
+                label="Select sensitivities",
+                options=[{"label": r, "value": r} for r in self.sensnames],
+                value=self.sensnames[0],
+                multi=True,
+            ),
+            html.Button(
+                id=self.register_component_unique_id(ViewSettings.IDs.RESET_BUTTON),
+                title="Reset Selected Sesitivities",
+                style={
+                    "fontSize": "10px",
+                    "marginTop": "10px",
+                }
+                if self.allow_click
+                else {"display": "none"},
+                children="Clear selected",
+            ),
+            wcc.Checklist(
+                id=self.register_component_unique_id(ViewSettings.IDs.PLOT_OPTIONS),
+                label="Plot options",
+                options=self.plot_options,
+                value=[],
+                labelStyle={"display": "block"},
+            ),
+            wcc.Dropdown(
+                id=self.register_component_unique_id(ViewSettings.IDs.LABEL),
+                label="Label",
+                options=self.label_options,
+                value="detailed",
+                clearable=False,
+            ),
+        ]
+
+    def set_callbacks(self) -> None:
+        @callback(
+            Output(
+                self.get_store_unique_id(PlugInIDs.Stores.ViewSetttings.REFERENCE),
+                "data",
+            ),
+            Input(
+                self.component_unique_id(ViewSettings.IDs.REFERENCE).to_string(),
+                "value",
+            ),
+        )
+        def _set_reference(ref: str) -> str:
+            return ref
+
+        @callback(
+            Output(
+                self.get_store_unique_id(PlugInIDs.Stores.ViewSetttings.SCALE), "data"
+            ),
+            Input(
+                self.component_unique_id(ViewSettings.IDs.SCALE).to_string(), "value"
+            ),
+        )
+        def _set_scale(scale: str) -> str:
+            return scale
+
+        @callback(
+            Output(
+                self.get_store_unique_id(PlugInIDs.Stores.ViewSetttings.SENSITIVITIES),
+                "data",
+            ),
+            Input(
+                self.component_unique_id(ViewSettings.IDs.SENSITIVITEIS).to_string(),
+                "value",
+            ),
+        )
+        def _set_sensitivities(sens: List[str]) -> List[str]:
+            return sens
+
+        @callback(
+            Output(
+                self.get_store_unique_id(PlugInIDs.Stores.ViewSetttings.RESET), "data"
+            ),
+            Input(
+                self.component_unique_id(ViewSettings.IDs.RESET_BUTTON).to_string(),
+                "n_clicks",
+            ),
+        )
+        def _set_button(n_clicks: int) -> int:
+            return n_clicks
+
+        @callback(
```

Comment:
This roundtrip is not necessary and should be removed.
I.e. use the component directly in the relevant callback.